self-core-impl/src/main/java/com/selfxdsd/core/tasks/ContractTasks.java

```diff
@@ -63,13 +71,24 @@ public Task register(final Issue issue) {
     @Override
     public Tasks ofProject(final String repoFullName,
                            final String repoProvider) {
-        throw new UnsupportedOperationException("Not implemented yet");
+        final List<Task> ofProject = this.tasks.stream()
+            .filter(t -> t.project().repoFullName().equals(repoFullName)
+                && t.project().provider().equals(repoProvider))
+            .collect(Collectors.toList());
+        return new ProjectTasks(repoFullName, repoProvider, ofProject,
+            this.storage);
     }
 
     @Override
     public Tasks ofContributor(final String username,
                                final String provider) {
-        throw new UnsupportedOperationException("Not implemented yet");
+        final List<Task> ofContributor = this.tasks.stream()
+            .filter(t -> t.assignee() != null
+                && t.assignee().username().equals(username)
+                && t.assignee().provider().equals(provider))
+            .collect(Collectors.toList());
+        return new ContributorTasks(username, provider, ofContributor,
+            this.storage);
```

Comment:
@criske Since these are already the Tasks of a Contract, I think in this method you should just check if the Contributor ID (username + prov) are matching the Contributor of the Contract. If yes, return ``this``, otherwise throw an exception ("Already seeing the Tasks of Contributor X, can't find other Tasks here").

Makes sense? :D